### PR TITLE
 Remove attribute-style accesses to (Geo)DataFrames columns and xarray Datasets variables and attributes within tutorials

### PR DESCRIPTION
**Pull request by spjuhel**
See also #939 for the codebase counterpart.

Changes proposed in this PR:
- Removal of attribute style-accesses to columns of pandas DataFrames, GeoDataFrames, and variables and attributes of xarray DataSets and DataArray within the documentation

This PR fixes SCRUM userstory 100: Avoid attribute-style access to DataFrame columns (https://tree.taiga.io/project/emanuel-schmid-climada/us/100).

### PR Author Checklist

- [x] Read the [Contribution Guide][contrib]
- [x] Correct target branch selected (if unsure, select `develop`)
- [x] Descriptive pull request title added
- [x] Source branch up-to-date with target branch
- [x] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
- [x] [Tests][testing] updated
- [x] [Tests][testing] passing
- [x] No new [linter issues][linter]
- [ ] [Changelog](https://github.com/CLIMADA-project/climada_python/blob/develop/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Review.html#reviewer-checklist) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/develop/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Testing.html
[linter]: https://climada-python.readthedocs.io/en/latest/guide/Guide_continuous_integration_GitHub_actions.html#static-code-analysis

### CODE REVIEW

**Review by luseverin — CHANGES_REQUESTED** [review]
Hey! Unfortunately I cannot open none of the modified tutorial files, as the notebooks seem to be broken (see the following error message, here for util_calibrate):
`Expected ',' or ']' after array element in JSON at position 133145 (line 1692 column 57)`
Apparently you removed one ',' or ']' when you modified the notebooks? You might want to check that out.

**Comment by spjuhel** [comment]
Woups, problems with the types of quotes used. Should be fixed now, sorry about that.

File: doc/tutorial/climada_engine_Impact.ipynb [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "| event_id |list(int)| id (>0) of each hazard event (Hazard.event_id)|\n",
     "| event_name |(list(str))| name of each event (Hazard.event_name)|\n",
     "| date |np.array| date of events (Hazard.date)|\n",
-    "| coord_exp |np.array| exposures coordinates [lat, lon] (in degrees) (Exposure.gdf.latidues, Exposure.gdf.longitude)|\n",
+    "| coord_exp |np.array| exposures coordinates [lat, lon] (in degrees) (Exposure.gdf['latitudes'], Exposure.gdf['longitude'])|\n",
```
Comment (luseverin) [inline doc/tutorial/climada_engine_Impact.ipynb:71]: I guess it should be 'latitude' here instead of 'latitudes'?

**Review by luseverin — COMMENTED** [review]
I looked at your changes, it looks good now. There is still one typo to be fixed though.

Otherwise, I think I spotted a couple missed instances of attribute-style accesses:
In `climada_hazard_TropCyclone.ipynb`:
`1753:     "print('Daily max sustained wind:', tc_syn.max_sustained_wind.groupby('time.day').max())"
`
and in `climada_util_calibrate.ipynb`:
`1042:     "emdat_subset = emdat[(emdat.cal_region2 == \"NA1\") & (emdat.year >= 2010)]\n",`

This aside, it looks all good, well done!

File: doc/tutorial/climada_engine_Impact.ipynb (hunk 1 above)
Comment (spjuhel) [inline doc/tutorial/climada_engine_Impact.ipynb:71]:
```suggestion
    "| coord_exp |np.array| exposures coordinates [lat, lon] (in degrees) (Exposure.gdf['latitude'], Exposure.gdf['longitude'])|\n",
```

Comment (spjuhel) [reply doc/tutorial/climada_engine_Impact.ipynb:71]: Done, good catch.

**Review by luseverin — APPROVED** [review]

**Pull request merged**